client/my-sites/email/email-providers-in-depth-comparison/comparison-table/index.tsx

```diff
@@ -1,21 +1,27 @@
-import { FunctionComponent } from 'react';
-import type { TranslateResult } from 'i18n-calypso';
+/* eslint-disable wpcalypso/jsx-classname-namespace */
```

Comment:
I disabled this rule to be able to keep class names like `email-providers-in-depth-comparison-table` instead of just `comparison-table`. The former is more explicit and also there is no risk of collision.

Reply:
Can you not use `email-providers-in-depth-comparison__comparison-table` for the class name>?